github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-periodics.yaml

```diff
@@ -65,3 +65,42 @@ periodics:
       - --repo=cluster-network-addons-operator
       - --skip_results_before_start_of_report=false
       - --pr_base_branch=main
+- name: periodic-cnao-workflow-k8s-nightly
+  cron: "2 7 * * *"
+  decorate: true
+  annotations:
+    testgrid-create-test-group: "false"
```

Comment:
@nunnatsa @RamLavi sorry for being late to the party.

Since you are not adding this periodic to the testgrid, as we do with most of the periodics (i.e. https://testgrid.k8s.io/kubevirt-periodics) you need to manually check the results in prow.

https://prow.ci.kubevirt.io/?job=periodic-cnao-workflow-k8s-nightly

I'd suggest either adding them to testgrid or adding email config to get notified in case of error.

Reply:
> @nunnatsa @RamLavi sorry for being late to the party.
> 
> Since you are not adding this periodic to the testgrid, as we do with most of the periodics (i.e. https://testgrid.k8s.io/kubevirt-periodics) you need to manually check the results in prow.
> 
> https://prow.ci.kubevirt.io/?job=periodic-cnao-workflow-k8s-nightly
> 
> I'd suggest adding email config to get notified in case of error. Or creating another testgroup in gen-config (see below) and adding them to testgrid group.

I'm not familiar with Thai, will be sure to read up on it, but as a primal question - Won't I see this job on the flake finder (which was my main goal) ?